pkg/types/types.go

```diff
@@ -160,14 +168,17 @@ type NFS struct {
 }
 
 type Ldap struct {
-	Disabled   bool   `yaml:"disabled,omitempty"`
-	Host       string `yaml:"host,omitempty"`
-	Port       string `yaml:"port,omitempty"`
-	Username   string `yaml:"username,omitempty"`
-	Password   string `yaml:"password,omitempty"`
-	Domain     string `yaml:"domain,omitempty"`
-	AdminGroup string `yaml:"adminGroup,omitempty"`
-	BindDN     string `yaml:"dn,omitempty"`
+	Disabled         bool   `yaml:"disabled,omitempty"`
+	Host             string `yaml:"host,omitempty"`
+	Port             string `yaml:"port,omitempty"`
+	Username         string `yaml:"username,omitempty"`
+	Password         string `yaml:"password,omitempty"`
+	Domain           string `yaml:"domain,omitempty"`
+	AdminGroup       string `yaml:"adminGroup,omitempty"`
+	UserDN           string `yaml:"userDN,omitempty"`
```

Comment:
Can we add comments on these fields with examples so they show up in the docs?

Reply:
Done